doc/configtables/lines.csv

```diff
@@ -1,6 +1,6 @@
 ,Unit,Values,Description
-types,--,"Values should specify a `line type in PyPSA <https://pypsa.readthedocs.io/en/latest/components.html#line-types>`_ in case AC lines or a custom linetype in case of DC lines modeled as Lines. Keys should specify the corresponding voltage level (e.g. 220., 300. and 380. kV)", "Specifies line types to assume for the different voltage levels of the ENTSO-E grid extraction. Should normally handle voltage levels 220, 300, and 380 kV."
-dc_type, --, "Values should specify a `line type in PyPSA <https://pypsa.readthedocs.io/en/latest/components.html#line-types>`_ for DC-lines.", "Specifies DC-line types."
+ac_types,--,"Values should specify a `line type in PyPSA <https://pypsa.readthedocs.io/en/latest/components.html#line-types>`_ for AC lines. Keys should specify the corresponding voltage level (e.g. 220., 300. and 380. kV)", "Specifies line types to assume for the different voltage levels of the ENTSO-E grid extraction. Should normally handle voltage levels 220, 300, and 380 kV."
```

Comment:
Ahh, sorry, have forgotten to add a comment: probably, we could also remove mentioning of ENTSO-E or add an explanation that it relates to Europe only. What do you think?

Reply:
Done, agree :) Soon I'll then merge